ui/app/appointments/services/calendarViewPopUp.js

```diff
@@ -2,15 +2,21 @@
 
 angular.module('bahmni.appointments')
     .service('calendarViewPopUp', ['$rootScope', 'ngDialog', '$state', '$translate', 'appointmentsService',
-        'confirmBox', 'checkinPopUp', 'appService', 'messagingService',
-        function ($rootScope, ngDialog, $state, $translate, appointmentsService, confirmBox, checkinPopUp, appService, messagingService) {
+        'confirmBox', 'checkinPopUp', 'appService', 'messagingService', 'appointmentCommonService',
+        function ($rootScope, ngDialog, $state, $translate, appointmentsService, confirmBox, checkinPopUp, appService, messagingService, appointmentCommonService) {
             var calendarViewPopUp = function (config) {
                 var popUpScope = $rootScope.$new();
                 var dialog;
                 var scope = config.scope;
+                var maxAppointmentProviders = appService.getAppDescriptor().getConfigValue('maxAppointmentProviders') || 1;
+                var appointmentProvider = scope.appointments[0].providers;
```

Comment:
use plural, its not a single provider. use appointmentProviders or just providers.

Reply:
Done.